PNlib/package.mo

```diff
@@ -1,5 +1,5 @@
 package PNlib
   annotation(
     uses(Modelica(version = "4.0.0")),
-    version = "2.2");
+    version = "3.0.0");
```

Comment:
Add license annotation to link to BSD 3 license.
See https://specification.modelica.org/master/annotations.html#licensing